assets/javascript/guides-version-dropdown.js

```diff
@@ -1,11 +1,11 @@
 $(document).ready(function() {
     $('#guidesindex-version-dropdown').on('change', function() {
-      var guideUrl = window.location.href.substring(window.location.href.indexOf("/guides/"));
-  
+      var url;
       if (this.value == 'latest') {
-        window.location.href = '/guides/';
+        url = '/guides/';
       } else {
-        window.location.href = '/version/' + this.value + guideUrl;
+        url = '/version/' + this.value + '/guides/';
```

Comment:
To explain above comment: This seems to be used in both getting started and guides.

Reply:
I removed the version selector from Getting Started in the `version-230` branch, because I don't think we really want to bother explaining how to install prior unsupported versions.

Reply:
Okayy sounds good!